### ISSUE TITLE
Prevent logging events on another date than announced 

### ISSUE THREAD

**Issue by eddiemuc** [cgeo/cgeo#16428]
Ticket #647240
C:geo version 2024.12.03-RC

A user on support requests that c:geo forbids logging an event attendance for a different date than the one where the event happens.

I wonder if there would be any corner cases where this leads to problems? Maybe only implement a warning pop-up instead of forbidden this completely?

**Comment by SammysHP** [comment]
A warning might be helpful. Sometimes logging on another date might be required (are there multi day events?).

**Comment by rouroni1** [comment]
Yes. Mega events usually are multi day... So that's one case where logging on a different must be allowed.

**Comment by ztNFny** [comment]
At least for GC it isn't possible to log it on another date anyway

**Comment by ztNFny** [comment]
At least for GC it isn't possible to log it on another date anyway, all logs are automatically changed to the events date on server side.

For other platforms: If the platform allows logging an event on any date why should c:geo restrict that? I'd consider that restriction a bug.

**Comment by moving-bits** [comment]
A "won't fix", IMHO
